src/main/java/frc/robot/Constants.java

```diff
@@ -19,4 +19,21 @@ public static class OperatorConstants
     {
         public static final int DRIVER_CONTROLLER_PORT = 0;
     }
+
+    public static class VisionConstants
+    {
```

Comment:
Very, very minor thing, but the general syntax in Java is to not have a newline before the curly brace

Reply:
You should be able to fix all at once by running:
CMD + OPTION + L on Mac
or
CTRL + ALT + L on Windows